docs/vocascan-server/cli.md

```diff
@@ -0,0 +1,121 @@
+
+### admin
+
+manage admin resources
+
+#### user
+
+manage vocascan users
+
+**Subcommands:**
+
+##### create
+
+Creates a vocascan-user in the configured database.
+
+- `-u`, `--username <username>` - username for the new user (required)
+- `-e`, `--email <email>` - email for the new user (required)
+- `-p`, `--password <password>` - password for the new user. If not set, a password of length `48` will be generated and
+  printed.
+- `-r`, `--role <name>` - chose a role for the new user. By default there are the following roles `admin`, `user`.
+
+**Examples**
+
+- `vocascan-server admin user create -u admin -e admin -r admin` - will create a user called `admin`, with their email
+  `admin`, with the `admin` role and a random password.
+- `vocascan-server admin user create -u hans -e hans@gmail.com -r user -p abcdefg` - will create a user called `hans`,
+  with their email `hans@gmail.com`, with the `user` role and a password `abcdefg`.
+
+!> it does not have to be a real email, it's only a login name with which you can log in to vocascan-server
+
+##### list
+
+Lists all users registered in the configured database.
+
+- `-r`, `--role <name>` - filter for a specific role name.
+
+**Examples**
+
+- `vocascan-server admin user list --role admin` - will only print users with the admin role
+- `vocascan-server admin user list` - will print all users registered in your database
+
+##### update
+
+Update a vocascan-user
+
+- `-i`, `--id <id>` - id to identify the user (one of `--id` or `--username` is required)
+- `-u`, `--username <username>` - username to identify the user (one of `--id` or `--username` is required)
+- `-p`, `--password <password>` - new password (don't set if you don't want to change the user's password)
+- `-r`, `--role <name>` - new role (don't set if you don't want to change the user's role)
+
+**Examples**
+
+- `vocascan-server admin user update -u admin -p Passw0rd` - will set `Passw0rd` as a new password for the admin user
+- `vocascan-server admin user update -i bd9c643e-c820-4b0c-bd04-f2d93f4e4dcb -r admin` - will give the user with the id
+  `bd9c643e-c820-4b0c-bd04-f2d93f4e4dcb` the `admin` role
+
+##### delete
+
+Deletes a vocascan-user
+
+- `-i`, `--id <id>` - id to identify the user (one of `--id` or `--username` is required)
+- `-u`, `--username <username>` - username to identify the user (one of `--id` or `--username` is required)
+- `-y`, `--yes` - Confirm directly that you want to delete that user. If this flag is not set, you will be interactively
+  prompted to confirm that operation
+
+**Examples**
+
+- `vocascan-server admin user delete -u hans` - will completely delete the user named `hans` after your interactive
+  confirm
+- `vocascan-server admin user delete -i bd9c643e-c820-4b0c-bd04-f2d93f4e4dcb -y` - will completely delete the user with
+  the id `bd9c643e-c820-4b0c-bd04-f2d93f4e4dcb` without confirm (you set `--yes`)
+
+### generate
+
+generation functions for strings needed for config files
```

Comment:
Maybe we could use “in config files“, so that we don't have the duplicate for in the sentence